proxy.js

```diff
@@ -29,4 +36,8 @@ exports.callEndpoint = function(appId, appSecret, endpoint, query, token, callba
   function generateHash(str) {
     return new Hashes.SHA256().hex(str);
   }
+
+  function createError(errMsg) {
+    callback(new Error('Error while trying to call platform endpoint ' + endpoint + errMsg));
```

Comment:
I think we should convert this to ES6:
`Error while trying to call platform endpoint ${endpoint}. ${erroMsg}`
WDYT?